motr/linux_kernel/m0tr_main.c

```diff
@@ -38,6 +39,9 @@ M0_INTERNAL int __init motr_init(void)
 	const struct module *m;
 	M0_THREAD_ENTER;
 
+#ifndef CONFIG_X86_64 //aarch64
```

Comment:
Why not to check explicitly for AARCH64?

Reply:
Why is this comment resolved? I don't see how it was resolved.

Reply:
Fixed by adding code for explicit check for aarch64 platform